### DISCUSSION TITLE
Trying to read exif information from CR2 or CR3 files. It returns null.

### DISCUSSION THREAD

**Discussion by mirceaion** [dlemstra/Magick.NET#1634] [Help]
I am using a .Net 8 console application and c# for instantiating a MagickImage. It takes a bit of time per file, like a couple of seconds, probably because files are 20MB and over.
Then I am calling GetExifProfile() and without fail, for each and every file, it returns null.
I used Canon's software to confirm that there is Exif info in those files.
I have to be honest, as there were way too many options, I allowed Visual Studio to recommend me which NuGet package to install when it noticed the unresolved MagickImage reference first time. It installed: Magick.NET-Q16-AnyCPU

Am I doing something wrong?

**Comment by dlemstra** [comment]
It think that there is currently no support for reading exif data in `cr2/cr3` files. This might be added in the future but that is why you are not getting any exif data.
